.github/CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# Documentation/Infrastructure
+/docs                                       @justinahinon @felixarntz
+/admin                                      @justinahinon @felixarntz
+/bin                                        @justinahinon @felixarntz
+/.github                                    @justinahinon @felixarntz
+
+# Focus: Images
+/modules/webp-uploads                       @adamsilverstein
+/tests/modules/webp-uploads                 @adamsilverstein
+
+# Focus: JavaScript
+
+# Focus: Object Caching
+
+# Focus: Measurement
+
+# Focus: Site Health
+/modules/audit-enqueued-assets				@manuelRod @audrasjb
+/tests/modules/audit-enqueued-assets		@manuelRod @audrasjb
```

Comment:
Should we add a line to `.editorconfig` to tackle this and force `indent_style = space` for this file?

Reply:
@tillkruss like this? https://github.com/WordPress/performance/pull/56/commits/c1a077075a88a4d13203833e6825f1f715599633